lib/output-marshaller.js

```diff
@@ -1,22 +1,15 @@
 const {Transform} = require('stream');
 const logger = require('util').debuglog('riff');
 const RiffError = require('./riff-error');
-const {canMarshall, determineContentTypes, marshaller} = require('./content-negotiation');
+const {marshaller} = require('./content-negotiation');
 const {AbstractMessage, Message} = require('@projectriff/message');
 
 module.exports = class OutputMarshaller extends Transform {
 
     constructor(index, contentType, options) {
         super(options);
-        if (index < 0) {
-            throw new RiffError('error-output-index-invalid', `invalid output index: ${index}`);
-        }
-        const acceptedContentType = determineContentTypes(contentType).accept;
-        if (!canMarshall(acceptedContentType)) {
```

Comment:
moved to the streaming pipeline, so a marshaller instance does not need to validate anything now